src/upload-report/upload-report.tsx

```diff
@@ -58,6 +64,18 @@ const UploadReport: React.FC<UploadReportProps> = ({
       )
       if (uploadFileResponse.ok) {
         const url = uploadFileResponse.data.url
+        const isPendingOrderInPayload =
```

Comment:
SRP violation

Reply:
has been taken care of in the latest commits